Enhancing custom permissions (#2152)
hashicorp/web-unified-docs: content/terraform-docs-common/docs/cloud-docs/users-teams-organizations/permissions/organization.mdx

| [Manage policy overrides](#manage-policy-overrides) | Team members can override failed **Soft mandatory** policy evaluations in all workspaces. |

### No policy overrides 

Teams can’t override failed **Soft mandatory** policy evaluations. 
 
### Delegate policy overrides 

Allow project and workspace managers to grant override permissions for **Soft mandatory** policy evaluations. When this setting is enabled, overriding failed policy evaluations is disabled by default. Project and workspace managers must manually enable the **Allow policy overrides** setting in their projects and workspaces. 
 
Refer to the [projects](/terraform/cloud-docs/users-teams-organizations/permissions/project#policy-overrides) and [workspaces](/terraform/cloud-docs/users-teams-organizations/permissions/workspace#policy-overrides) team permission references for more information. 
 
### Mange policy overrides 

> Team members can override failed **Soft mandatory** policy evaluations in all workspaces. 

Author: ```suggestion
Team members can override failed **Soft mandatory** policy evaluations in all workspaces. 

This setting also gives teams read access to all workspaces in the organization. To prevent read access, enable the [**Delegate policy overrides**](#delegate-policy-overrides) setting instead.
```

<!-- END: TFC:only name:public-registry -->

## Policy overrides

Policy override settings only apply to policies that have a **Soft mandatory** enforcement level. Refer to [Policy enforcement levels](/terraform/cloud-docs/policy-enforcement/manage-policy-sets#policy-enforcement-levels) for more information.

| Permission name | Description |
| --- | --- |
| [No policy overrides](#no-policy-overrides) | Teams can’t override failed **Soft mandatory** policy evaluations. |

> | [Delegate policy overrides](#delegate-policy-overrides) | Allow project and workspace managers to grant override permissions for **Soft mandatory** policy evaluations. When this setting is enabled, the ability to override failed policy evaluations is disabled by default. Project and workspace managers must manually enable the **Allow policy overrides** setting in their projects and workspaces.  |

Reviewer: I wonder if we need to tweak this sentence a bit:

> When this setting is enabled, the ability to override failed policy evaluations is disabled by default.

This setting could activate an existing permission for a project admin, for example -- they already have the project-level half of the permission required to override soft mandatory policies on those workspaces.

Maybe something like:

```suggestion
| [Delegate policy overrides](#delegate-policy-overrides) | Allow project and workspace managers to grant override permissions for **Soft mandatory** policy evaluations. When this setting is enabled for a team, its members can override failed policy evaluations on projects and workspaces that they manage. |
```
